server/IDEs/eclipse/Refine with extensions.launch

```diff
@@ -1,18 +1,18 @@
+    <booleanAttribute key="org.eclipse.jdt.launching.ATTR_USE_CLASSPATH_ONLY_JAR" value="false"/>
+    <stringAttribute key="org.eclipse.jdt.launching.CLASSPATH_PROVIDER" value="org.eclipse.m2e.launchconfig.classpathProvider"/>
+    <stringAttribute key="org.eclipse.jdt.launching.MAIN_TYPE" value="com.google.refine.Refine"/>
+    <stringAttribute key="org.eclipse.jdt.launching.PROJECT_ATTR" value="server"/>
+    <stringAttribute key="org.eclipse.jdt.launching.SOURCE_PATH_PROVIDER" value="org.eclipse.m2e.launchconfig.sourcepathProvider"/>
+    <stringAttribute key="org.eclipse.jdt.launching.VM_ARGUMENTS" value="-Drefine.autoreloading=true -Dbutterfly.modules.path=modules,&quot;${workspace_loc:extensions}&quot;,&quot;${workspace_loc:jython}&quot;,&quot;${workspace_loc:extensions}&quot;,&quot;${workspace_loc:database}&quot;"/>
+    <stringAttribute key="org.eclipse.jdt.launching.WORKING_DIRECTORY" value="${workspace_loc:OpenRefine}"/>
```

Comment:
Those look like unrelated change, right?

Reply:
Yes @wetneb , these are un related and they got no effect.

Reply:
Then we should make sure this pull request leaves this file untouched.